src/Bowling.js

```diff
@@ -0,0 +1,112 @@
+var Bowling = function(){
+  this.rollsArray = [];
+  this.scoresArray = [];
+  this.rollCounter = 0;
+  this.frameCounter = 1;
+};
+
+Bowling.prototype.roll = function(rollScore){
+  var self = this;
+
+  function throwAlert(){
+    alert("Illegal score");
+  };
```

Comment:
Don't alert in your business logic - throw an error and handle it at the presentation layer. Or don't throw an error! Handle it gracefully